I am still seeing off-grid issues with this unfortunately, it seems. As well as the pin issue all the layers above `li` changed direction from before and I think that's (part of, at least) the problem.

example errors:
```
[ERROR] In topology of <id:1918884 ContactTerminal core.abc_60546_new_n26809 li_mcon_m1 [905.28um 2313.915um] 0.17um x 0.17um -TD---->
        A: &<id:312170 RoutingPad core.abc_60546_new_n26809 [904.89um 2313.915um] <Occurrence <id:201224 Cell corona>:core.subckt_19789_nand2_x0:<id:6739 Vertical i1 li [1.11um 1.37um] [1.11um 6.46um] 0.17um>>>
        L: &<id:1918886 Vertical core.abc_60546_new_n26809 m1 [905.28um 2313.915um] [905.28um 2314.38um] 0.32um rpD:0 -X--UC---TD-----i-bt-->
        Terminal vertical segment X 905.28um axis is outside RoutingPad [904.89um 904.89um].
[ERROR] In topology of <id:1897599 ContactTerminal core.soc_cpu_minerva_payload_src2_44(25) li_mcon_m1 [936.56um 2355um] 0.17um x 0.17um -TD---->
        A: &<id:337742 RoutingPad core.soc_cpu_minerva_payload_src2_44(25) [936.255um 2355um] <Occurrence <id:201224 Cell corona>:core.subckt_13256_inv_x0:<id:5626 Vertical i li [0.255um 1.37um] [0.255um 8.63um] 0.17um>>>
        L: &<id:1897601 Vertical core.soc_cpu_minerva_payload_src2_44(25) m1 [936.56um 2355um] [936.56um 2354.16um] 0.32um rpD:0 -X--UC---TD-----i-bt-->
        Terminal vertical segment X 936.56um axis is outside RoutingPad [936.255um 936.255um].
[ERROR] In topology of <id:1920819 ContactTerminal core.abc_60546_new_n26665 li_mcon_m1 [893.78um 2073.665um] 0.17um x 0.17um -TD---->
        A: &<id:292889 RoutingPad core.abc_60546_new_n26665 [894.11um 2073.665um] <Occurrence <id:201224 Cell corona>:core.subckt_19703_nand4_x0:<id:7019 Vertical i1 li [1.11um 1.37um] [1.11um 5.96um] 0.17um>>>
        L: &<id:1920821 Vertical core.abc_60546_new_n26665 m1 [893.78um 2073.665um] [893.78um 2074.68um] 0.32um rpD:0 -X--UC---TD-----i-bt-->
        Terminal vertical segment X 893.78um axis is outside RoutingPad [894.11um 894.11um].
[ERROR] In topology of <id:1908669 ContactTerminal core.abc_60546_new_n26664 li_mcon_m1 [895.16um 2073.665um] 0.17um x 0.17um -TD---->
        A: &<id:293400 RoutingPad core.abc_60546_new_n26664 [894.74um 2073.665um] <Occurrence <id:201224 Cell corona>:core.subckt_19703_nand4_x0:<id:7030 Vertical i2 li [1.74um 1.37um] [1.74um 5.96um] 0.17um>>>
        L: &<id:1908671 Vertical core.abc_60546_new_n26664 m1 [895.16um 2073.665um] [895.16um 2076.21um] 0.32um rpD:0 -X--UC---TD-----i-bt-->
        Terminal vertical segment X 895.16um axis is outside RoutingPad [894.74um 894.74um].
```

Layers in the new library with this fix:
```python
    rg = CRL.RoutingGauge.create('StdCellLib')
    rg.setSymbolic(False)
    metal = tech.getLayer('li')
    via = tech.getLayer('li_mcon_m1')
    setEnclosures(via, metal, (u(0.0), u(0.0)))
    rg.addLayerGauge(CRL.RoutingLayerGauge.create(
        metal, CRL.RoutingLayerGauge.Horizontal, CRL.RoutingLayerGauge.PinOnly, 0, 0.0,
        u(0.0), u(0.5), u(0.33), u(0.17), u(0.17), u(0.17),
    ))
    metal = tech.getLayer('m1')
    via = tech.getLayer('li_mcon_m1')
    setEnclosures(via, metal, (u(0.06), u(0.03)))
    via = tech.getLayer('m1_via_m2')
    setEnclosures(via, metal, (u(0.085), u(0.055)))
    rg.addLayerGauge(CRL.RoutingLayerGauge.create(
        metal, CRL.RoutingLayerGauge.Vertical, CRL.RoutingLayerGauge.Default, 1, 0.0,
        u(0.0), u(0.46), u(0.32), u(0.23), u(0.15), u(0.14),
    ))
    metal = tech.getLayer('m2')
    via = tech.getLayer('m1_via_m2')
    setEnclosures(via, metal, (u(0.055), u(0.085)))
    via = tech.getLayer('m2_via2_m3')
    setEnclosures(via, metal, (u(0.04), u(0.085)))
    rg.addLayerGauge(CRL.RoutingLayerGauge.create(
        metal, CRL.RoutingLayerGauge.Horizontal, CRL.RoutingLayerGauge.Default, 2, 0.0,
        u(0.0), u(0.51), u(0.37), u(0.26), u(0.2), u(0.14),
    ))
    metal = tech.getLayer('m3')
    via = tech.getLayer('m2_via2_m3')
    setEnclosures(via, metal, (u(0.065), u(0.065)))
    via = tech.getLayer('m3_via3_m4')
    setEnclosures(via, metal, (u(0.09), u(0.06)))
    rg.addLayerGauge(CRL.RoutingLayerGauge.create(
        metal, CRL.RoutingLayerGauge.Vertical, CRL.RoutingLayerGauge.Default, 3, 0.0,
        u(0.0), u(0.68), u(0.38), u(0.33), u(0.2), u(0.3),
    ))
```
etc

in the old library:
```python
rg = CRL.RoutingGauge.create('FlexLib')
    rg.setSymbolic(False)
    metal = tech.getLayer('li')
    via = tech.getLayer('li_mcon_m1')
    setEnclosures(via, metal, (u(0.0), u(0.0)))
    rg.addLayerGauge(CRL.RoutingLayerGauge.create(
        metal, CRL.RoutingLayerGauge.Horizontal, CRL.RoutingLayerGauge.PinOnly, 0, 0.0,
        u(0.0), u(0.51), u(0.17), u(0.17), u(0.17), u(0.17),
    ))
    metal = tech.getLayer('m1')
    via = tech.getLayer('li_mcon_m1')
    setEnclosures(via, metal, (u(0.03), u(0.06)))
    via = tech.getLayer('m1_via_m2')
    setEnclosures(via, metal, (u(0.055), u(0.085)))
    rg.addLayerGauge(CRL.RoutingLayerGauge.create(
        metal, CRL.RoutingLayerGauge.Horizontal, CRL.RoutingLayerGauge.Default, 1, 0.0,
        u(0.0), u(0.46), u(0.32), u(0.23), u(0.15), u(0.14),
    ))
    metal = tech.getLayer('m2')
    via = tech.getLayer('m1_via_m2')
    setEnclosures(via, metal, (u(0.085), u(0.055)))
    via = tech.getLayer('m2_via2_m3')
    setEnclosures(via, metal, (u(0.085), u(0.04)))
    rg.addLayerGauge(CRL.RoutingLayerGauge.create(
        metal, CRL.RoutingLayerGauge.Vertical, CRL.RoutingLayerGauge.Default, 2, 0.0,
        u(0.0), u(0.51), u(0.37), u(0.26), u(0.2), u(0.14),
    ))
    metal = tech.getLayer('m3')
    via = tech.getLayer('m2_via2_m3')
    setEnclosures(via, metal, (u(0.065), u(0.065)))
    via = tech.getLayer('m3_via3_m4')
    setEnclosures(via, metal, (u(0.06), u(0.09)))
    rg.addLayerGauge(CRL.RoutingLayerGauge.create(
        metal, CRL.RoutingLayerGauge.Horizontal, CRL.RoutingLayerGauge.Default, 3, 0.0,
        u(0.0), u(0.68), u(0.38), u(0.33), u(0.2), u(0.3),
    ))
```

--- a/thirdparty/open_pdk/C4M.Sky130/libs.tech/coriolis/techno/etc/coriolis2/node130/sky130/StdCellLib.py
+++ b/thirdparty/open_pdk/C4M.Sky130/libs.tech/coriolis/techno/etc/coriolis2/node130/sky130/StdCellLib.py
@@ -31,34 +31,34 @@ def _routing():
     via = tech.getLayer('li_mcon_m1')
     setEnclosures(via, metal, (u(0.0), u(0.0)))
     rg.addLayerGauge(CRL.RoutingLayerGauge.create(
-        metal, CRL.RoutingLayerGauge.Horizontal, CRL.RoutingLayerGauge.Default, 0, 0.0,
-        u(0.0), u(0.5), u(0.33), u(0.17), u(0.17), u(0.17),
+        metal, CRL.RoutingLayerGauge.Horizontal, CRL.RoutingLayerGauge.PinOnly, 0, 0.0,
+        u(0.0), u(0.51), u(0.17), u(0.17), u(0.17), u(0.17),
     ))
     metal = tech.getLayer('m1')
     via = tech.getLayer('li_mcon_m1')
-    setEnclosures(via, metal, (u(0.06), u(0.03)))
+    setEnclosures(via, metal, (u(0.03), u(0.06)))
     via = tech.getLayer('m1_via_m2')
-    setEnclosures(via, metal, (u(0.085), u(0.055)))
+    setEnclosures(via, metal, (u(0.055), u(0.085)))
     rg.addLayerGauge(CRL.RoutingLayerGauge.create(
-        metal, CRL.RoutingLayerGauge.Vertical, CRL.RoutingLayerGauge.Default, 1, 0.0,
+        metal, CRL.RoutingLayerGauge.Horizontal, CRL.RoutingLayerGauge.Default, 1, 0.0,
         u(0.0), u(0.46), u(0.32), u(0.23), u(0.15), u(0.14),
     ))
     metal = tech.getLayer('m2')
     via = tech.getLayer('m1_via_m2')
-    setEnclosures(via, metal, (u(0.055), u(0.085)))
+    setEnclosures(via, metal, (u(0.085), u(0.055)))
     via = tech.getLayer('m2_via2_m3')
-    setEnclosures(via, metal, (u(0.04), u(0.085)))
+    setEnclosures(via, metal, (u(0.085), u(0.04)))
     rg.addLayerGauge(CRL.RoutingLayerGauge.create(
-        metal, CRL.RoutingLayerGauge.Horizontal, CRL.RoutingLayerGauge.Default, 2, 0.0,
+        metal, CRL.RoutingLayerGauge.Vertical, CRL.RoutingLayerGauge.Default, 2, 0.0,
         u(0.0), u(0.51), u(0.37), u(0.26), u(0.2), u(0.14),
     ))
     metal = tech.getLayer('m3')
     via = tech.getLayer('m2_via2_m3')
     setEnclosures(via, metal, (u(0.065), u(0.065)))
     via = tech.getLayer('m3_via3_m4')
-    setEnclosures(via, metal, (u(0.09), u(0.06)))
+    setEnclosures(via, metal, (u(0.06), u(0.09)))
     rg.addLayerGauge(CRL.RoutingLayerGauge.create(
-        metal, CRL.RoutingLayerGauge.Vertical, CRL.RoutingLayerGauge.Default, 3, 0.0,
+        metal, CRL.RoutingLayerGauge.Horizontal, CRL.RoutingLayerGauge.Default, 3, 0.0,
         u(0.0), u(0.68), u(0.38), u(0.33), u(0.2), u(0.3),
     ))
     metal = tech.getLayer('m4')
@@ -67,14 +67,14 @@ def _routing():
     via = tech.getLayer('m4_via4_m5')
     setEnclosures(via, metal, (u(0.19), u(0.19)))
     rg.addLayerGauge(CRL.RoutingLayerGauge.create(
-        metal, CRL.RoutingLayerGauge.Horizontal, CRL.RoutingLayerGauge.Default, 4, 0.0,
+        metal, CRL.RoutingLayerGauge.Vertical, CRL.RoutingLayerGauge.Default, 4, 0.0,
         u(0.0), u(1.48), u(1.18), u(0.33), u(0.8), u(0.3),
     ))
     metal = tech.getLayer('m5')
     via = tech.getLayer('m4_via4_m5')
     setEnclosures(via, metal, (u(0.31), u(0.31)))
     rg.addLayerGauge(CRL.RoutingLayerGauge.create(
-        metal, CRL.RoutingLayerGauge.Vertical, CRL.RoutingLayerGauge.Default, 5, 0.0,
+        metal, CRL.RoutingLayerGauge.Horizontal, CRL.RoutingLayerGauge.Default, 5, 0.0,
         u(0.0), u(3.2), u(1.6), u(1.6), u(0.8), u(1.6),
     ))
     af.addRoutingGauge(rg)

--- a/thirdparty/open_pdk/C4M.Sky130/libs.tech/coriolis/techno/etc/coriolis2/node130/sky130/StdCellLib.py
+++ b/thirdparty/open_pdk/C4M.Sky130/libs.tech/coriolis/techno/etc/coriolis2/node130/sky130/StdCellLib.py
@@ -98,6 +98,7 @@ def _routing():
         cfg.etesian.uniformDensity = True
         cfg.etesian.routingDriven = False
         cfg.etesian.latchUpDistance = u(30.0 - 1.0)
+        cfg.etesian.diodeName = 'diode_w1'
         cfg.etesian.antennaInsertThreshold = 0.50
         cfg.etesian.antennaMaxWL = u(250.0)
         cfg.etesian.feedNames = 'tie,decap_w0'

